meson_options.txt

```diff
@@ -0,0 +1,6 @@
+option('py3modules_dir',
+    type : 'string',
+    value : '',
+    description: 'Where to install python3 modules'
```

Comment:
Is this option actually needed? For a while (since 0.60.0), Meson's python module has been handling dist-packages on Debian. It also has a `-Dpython.purelibdir=` option to specify exact paths, but the autodetection should work.

Not sure which minimum version of Meson is currently needed for Mint support (0.49.0 is a lot more backwards-compatible than Mint needs, IIRC).

Reply:
We need minimum 0.56.2 (What LMDE5 is on). Mint/Ubuntu have 0.61.2 currently.

0.61.1 is offered in debian backports, but we keep that a low priority which makes this version unavailable unless explicitly requested by the user. Probably best to accomodate 0.56.2, at least for now.

Reply:
Alright, that's a bit disappointing but hopefully only temporary. :) Makes sense to stick with the option for now.